halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/kubernetes/KubernetesAddAccountCommand.java

```diff
@@ -143,6 +149,14 @@ protected Account buildAccount(String accountName) {
     account.setCheckPermissionsOnStartup(checkPermissionsOnStartup);
     account.setLiveManifestCalls(liveManifestCalls);
     account.setCacheThreads(cacheThreads);
+    customResources.forEach(
+        customResource ->
+            account
+                .getCustomResources()
+                .add(
+                    new KubernetesAccount.CustomKubernetesResource()
+                        .setKubernetesKind(customResource)
+                        .setVersioned(false)));
```

Comment:
Was unable to verify this property on the custom resources is actually used anywhere. Need guidance on how to properly setup the command for accepting a list of custom resources _and_ whether they are versioned.

Reply:
Hmm, I'm not entirely sure where that's done, either.  But the rest of the code looks fine.  😄